extensions/wikia/ImageReview/modules/PromoteImage/PromoteImageReviewHelper.class.php

```diff
@@ -117,15 +117,15 @@ protected function saveStats($statsInsert, $sqlWhere, $action) {
 				case '':
 					//	$stats['reviewer'] += count($images);
 					//	$stats['unreviewed'] -= count($images);
-					$stats['questionable'] += count($sqlWhere[ImageReviewStatuses::STATE_QUESTIONABLE]);
+					$stats['questionable'] += count($sqlWhere[ImageReviewStates::QUESTIONABLE]);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "questionable" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)